pyproject.toml

```
@@ -1,3 +1,8 @@
 [build-system]
 requires = ["setuptools >= 40.9.0", "wheel"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+markers = [
+    "mlmodel: marks tests as (slow) ml models (deselect with '-m \"not ml\"')",
```

Whatever's after the colon is an optional description but good to stick with the correct naming convention in the script

```suggestion
    "mlmodel: marks tests as (slow) ml models (deselect with '-m "not mlmodel"')",
```
https://docs.pytest.org/en/6.2.x/mark.html

---

Good catch